packages/sdk-react/src/contexts/TurnkeyContext.tsx

```diff
@@ -1,31 +1,45 @@
+"use client";
+
 import { ReactNode, createContext, useState, useEffect, useRef } from "react";
 import {
   Turnkey,
   TurnkeyIframeClient,
   TurnkeyPasskeyClient,
   TurnkeySDKBrowserConfig,
   TurnkeyBrowserClient,
+  TurnkeyWalletClient,
+  AuthClient,
 } from "@turnkey/sdk-browser";
+import type { WalletInterface } from "@turnkey/wallet-stamper";
+import { useUserSession } from "../hooks/use-session";
 
 export interface TurnkeyClientType {
+  client: TurnkeyBrowserClient | undefined;
   turnkey: Turnkey | undefined;
   authIframeClient: TurnkeyIframeClient | undefined;
   passkeyClient: TurnkeyPasskeyClient | undefined;
+  walletClient: TurnkeyWalletClient | undefined;
   getActiveClient: () => Promise<TurnkeyBrowserClient | undefined>;
 }
 
 export const TurnkeyContext = createContext<TurnkeyClientType>({
+  client: undefined,
```

Comment:
if this is the active client, should we name it `activeClient` for consistency with `getActiveClient`?

Reply:
yeah that makes sense! I was thinking of simplifying the name to abstract away some of the complexity of the multiple clients